src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -89,7 +101,41 @@ function Restore() {
         </SafeAreaView>
       )
     }
-    // TODO(ACT-781): Implement Success screen
+    case KeylessBackupStatus.Completed: {
+      return (
+        <SafeAreaView style={styles.container}>
+          <View style={styles.finishedContainer}>
+            <View style={styles.iconContainer}>
+              <GreenLoadingSpinnerToCheck />
+            </View>
+            <Text style={styles.title}>{t('keylessBackupStatus.restore.completed.title')}</Text>
+            <Text style={styles.body}>
+              {totalBalanceLocal.gt(0)
+                ? t('keylessBackupStatus.restore.completed.bodyBalance', {
+                    localCurrencySymbol,
+                    totalBalance: totalBalanceLocal.toFormat(2),
```

Comment:
should we use the TokenDisplay component for this instead of explicitly constructing this here? E.g., see `balanceAvailable` string